java-security/src/main/java/com/sap/cloud/security/token/validation/validators/JsonWebKeySet.java

```diff
@@ -44,4 +43,17 @@ public void putAll(JsonWebKeySet jsonWebKeySet) {
 				.filter(jwk -> algorithm.equals(jwk.getKeyAlgorithm()))
 				.filter(jwk -> kid.equals(jwk.getId()));
 	}
+
+	public boolean containsZoneId(String zoneId) {
```

Comment:
I didn't find unittest for the new methods. I think it would be helpful to have them

Reply:
yes . thanks!

Reply:
I've checked the code coverage and its called about 10 times and tested with accepted zoneIds and one, which does not exist.